crates/cheatcodes/spec/src/vm.rs

```diff
@@ -317,6 +317,10 @@ interface Vm {
     #[cheatcode(group = Evm, safety = Unsafe)]
     function roll(uint256 newHeight) external;
 
+    /// Gets the current `block.number`
+    #[cheatcode(group = Evm, safety = Unsafe)]
+    function getBlockNumber() external returns (uint256 height);
```

Comment:
```suggestion
    #[cheatcode(group = Evm, safety = Safe)]
    function getBlockNumber() external view returns (uint256 height);
```

Reply:
done in https://github.com/foundry-rs/foundry/pull/6630/commits/20e1e769aebdbe53d11d90059d7a6a883d27b4d2